pages/public_cloud/containers_orchestration/managed_private_registry/deploy-chart-from-registry-in-kubernetes/guide.de-de.md

```diff
@@ -18,182 +18,83 @@ This tutorial presupposes that you already have a working OVHcloud Managed Kuber
 
 You also need to have a working OVHcloud Managed Private Registry and have followed the guide on [managing Helm charts in the OVHcloud Managed Private Registry](/pages/public_cloud/containers_orchestration/managed_private_registry/using-helm-chart-museum).
 
-You should have at least one wordpress Helm chart in your Private Registry:
+You should have at least one Wordpress Helm chart in your Managed Private Registry:
```

Comment:
I think it can be useful to add a link to the tutorial that help the user to push/upload the wordpress helm chart in a private registry: https://help.ovhcloud.com/csm/en-ie-public-cloud-private-registry-helm-chart-museum?id=kb_article_view&sysparm_article=KB0050370

Reply:
The link is already resent l 19 just above, but I forgot to update the path